Marlin/Marlin_main.cpp

```diff
@@ -7225,8 +7227,58 @@ void quickstop_stepper() {
    *       S[bool]   Turns leveling on or off
    *       Z[height] Sets the Z fade height (0 or none to disable)
    *       V[bool]   Verbose - Print the leveling grid
+   *
+   *       L[index]  Load UBL mesh from index (0 is default)
    */
   inline void gcode_M420() {
+    #if ENABLED(AUTO_BED_LEVELING_UBL)
+      //
+      // Load a Mesh from the EEPROM
+      //
+
+      if (code_seen('L')) {     // Load Current Mesh Data
+        uint8_t Storage_Slot = code_has_value() ? code_value_int() : ubl.state.eeprom_storage_slot;
+
+        int k = E2END - sizeof(ubl.state);
+        int j = (k - ubl_eeprom_start) / sizeof(z_values);
+
+        if (Storage_Slot < 0 || Storage_Slot >= j || ubl_eeprom_start <= 0) {
+          SERIAL_PROTOCOLLNPGM("?EEPROM storage not available for use.\n");
+          return;
+        }
+        ubl.load_mesh(Storage_Slot);
+        ubl.state.eeprom_storage_slot = Storage_Slot;
+        if (Storage_Slot != ubl.state.eeprom_storage_slot)
```

Comment:
That was cut and pasted from some other code...   We can help the compiler out and simplify that for it.   Maybe it will return the favor and and a ; some time it notices one missing.